src/app/page.tsx

```diff
@@ -1,11 +1,15 @@
 import { Inter } from 'next/font/google'
+import Heading from './components/LargeHeading'
 
 const inter = Inter({ subsets: ['latin'] })
 
 export default function Home() {
   return (
     <main className="bg-red-600">
-      hello
+      {/* Only For Checking it, will be removed later  */}
+      <Heading>
```

Comment:
storybook is better way of representing individual components

Reply:
Okay I am gonna make a Task for implementing Storybook then.